frappe/oauth.py

```diff
@@ -11,7 +11,8 @@
 from oauthlib.oauth2.rfc6749.endpoints.revocation import RevocationEndpoint
 from oauthlib.common import Request
 
-separated_by = " "
+def oauth_url_delimiter(separator_character=" "):
+	return separator_character
```

Comment:
better choice for name would be `get_url_delimiter` since it is expected to return something.